var/spack/repos/builtin/packages/blitz/package.py

```diff
@@ -28,8 +28,9 @@
 class Blitz(AutotoolsPackage):
     """N-dimensional arrays for C++"""
     homepage = "http://github.com/blitzpp/blitz"
-    url = "https://github.com/blitzpp/blitz/tarball/1.0.0"
+    url = "https://github.com/blitzpp/blitz/archive/1.0.1.tar.gz"
```

Comment:
If you change the download URL, you need to change the checksum for 1.0.0 as well.